raritan/tests/test_duration.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+from raritan.decorators import _get_formatted_duration
+
+"""
+Test the duration formatting function.
+"""
+
+
+def test_formatting_some_durations() -> None:
```

Comment:
Thanks for this addition. I feel like that's a great function and it's nice to have a test on it.